ui/components/multichain-accounts/multichain-account-list/multichain-account-list.stories.tsx

```diff
@@ -0,0 +1,101 @@
+import { StoryObj, Meta } from '@storybook/react';
+import {
+  MultichainAccountList,
+  type MultichainAccountListProps,
+} from './multichain-account-list';
+import { AccountGroupId } from '@metamask/account-api';
+import { AccountTreeWallets } from '../../../selectors/multichain-accounts/account-tree.types';
+import { AccountWalletMetadata } from '@metamask/account-tree-controller';
+
+const mockWallets: AccountTreeWallets = {
```

Comment:
is there already a mock data object somewhere we can just import here? i always debate if its better to have independent mocks per test to ensure their individuality and just assume the code repetition or have one mocks consumed by multiple tests and ensure consistency